tracim/tracim/controllers/read.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
move the class ContentController to content.py and remove read.py
add a comment to ContentController in order to explain that it is the content controller in charge of mark_read feature only.